components/analytics/true_up_review.tsx

```diff
@@ -165,7 +165,7 @@ const TrueUpReview: React.FC = () => {
         >
             <FormattedMessage
                 id='admin.billing.trueUpReview.docsLinkCTA'
-                defaultMessage='Learn more about true-up here.'
+                defaultMessage='Learn more about true-up.'
```

Comment:
curious why did'nt we ran i18n-extract here

Reply:
Oh my bad! That change is still living in my local env. Good catch!